_plugins/cosmos_meta_tag.rb

```diff
@@ -96,25 +96,25 @@ def build_page(meta, page)
         end
         if data['example']
           page << "\nExample Usage:\n"
-          page << "<figure class=\"highlight\"><pre><code class=\"language-bash\" data-lang=\"bash\">"
-          page << "#{data['example'].gsub('<','&lt;').gsub('>','&gt;')}</code></pre></figure>\n"
+          page << "```ruby\n"
+          page << "#{data['example'].strip}\n"
+          page << "```\n"
         end
-        modifiers[keyword] = data['modifiers']
-      end
-      bump_level = false
-      modifiers.each do |keyword, modifiers|
-        if modifiers
-          unless @modifiers.values.include?(modifiers.keys)
+        saved_level = @level
+        if data['modifiers']
+          bump_level = false
+          unless @modifiers.values.include?(data['modifiers'].keys)
             if bump_level == false
               bump_level = true
               @level += 1
             end
-            @modifiers[keyword] = modifiers.keys
+            @modifiers[keyword] = data['modifiers'].keys
             page << "\n#{'#' * (@level - 1)} #{keyword} Modifiers\n"
             page << "The following keywords must follow a #{keyword} keyword.\n"
-            build_page(modifiers, page)
+            build_page(data['modifiers'], page)
           end
         end
+        @level = saved_level
```

Comment:
This change puts the modifiers right under the keyword instead of all together at the end. Mostly it doesn't matter because there's only one top level keyword in a section but the Plugin page is the notable exception.